src/System.Device.Gpio.Tests/RaspberryPiDriverTests.cs

```diff
@@ -52,5 +52,41 @@ public void InputPullResistorsWork()
                 Assert.Equal(PinValue.Low, controller.Read(OpenPin));
             }
         }
+
+        [Fact]
+        public void HighPulledPinDoesNotChangeToLowWhenChangedToOutput()
```

Comment:
Are there other drivers that also won't trigger to Low? for example libgpiod?

Reply:
Yes, libgpiod seems to work as well. I need to check why I didn't add the test there as well.